sdk/src/signature.rs

```diff
@@ -54,6 +55,21 @@ impl Signature {
     pub fn verify(&self, pubkey_bytes: &[u8], message_bytes: &[u8]) -> bool {
         self.verify_verbose(pubkey_bytes, message_bytes).is_ok()
     }
+
+    pub fn last_n_bits_are_zero(&self, n: u8) -> bool {
```

Comment:
that makes sense. since i need access to Signature's `GenericArray<u8, U64>` to verify ending bits, is it better to define a `get_bytes()` method or something in Signature, so we can validate these bits outside of the Signature class?

Reply:
You can just use: `signature.as_ref()`:
https://github.com/solana-labs/solana/blob/a2d987067/sdk/src/signature.rs#L75-L79

Reply:
oh that's nice. perfect. thank you